Lib/bdb.py

```diff
@@ -62,12 +104,28 @@ def trace_dispatch(self, frame, event, arg):
         return self.trace_dispatch
 
     def dispatch_line(self, frame):
+        """Trace function when a new line of code is executed.
```

Comment:
You're probably not going to like what I changed it to, but maybe it will help come up with something else.   :-)